packages/aws-cdk-lib/aws-autoscaling/test/scaling.test.ts

```diff
@@ -327,6 +327,62 @@ test('step scaling with evaluation period configured', () => {
   });
 });
 
+test('step scaling with evaluation period and data points to alarm configured', () => {
```

Comment:
```suggestion
describe('datapointsToAlarm', () => {
  test('step scaling with evaluation period and data points to alarm configured', () => {
    ...
```
Let's group these tests to keep it organized.